src/custom/theme/index.tsx

```diff
@@ -17,6 +22,30 @@ export enum Z_INDEX {
   tooltip = 1080,
 }
 
+type TextProps = TextPropsUni & { override?: boolean }
+
+export const TextWrapper = styled(Text)<{ color: keyof Colors; override?: boolean }>`
+  color: ${({ color, theme, override }) => {
+    const colour = (theme as any)[color]
```

Comment:
`color` or `colour`?
Pick one :)

I know, I know, there's already a `color` const. Then `themeColor`?

Reply:
this is just a mod of theme i didn't code this, can we leave it as is pretty please?

Reply:
ok ok